astroquery/tests/test_cache.py

```diff
@@ -181,13 +169,12 @@ def test_timeout(monkeypatch):
 
     # Testing a cache timeout of "none"
     cache_conf.cache_timeout = None
-    set_response(TEXT1)
 
     resp = mytest.test_func(URL1)
     assert resp.content == TEXT2  # cache is accessed
```

Comment:
I have unintentionally changed this part of the tests - it no longer guarantees that `TEXT2` is read from the cache. I'll address this tomorrow.

Reply:
I've added a line that deletes `requests.Session.request` for the remainder of this test function, so that the following `test_func()` call can succeed only if it accesses the cache.